spec/system/account_page_spec.rb

```diff
@@ -13,6 +13,7 @@
 
     if Rails.application.gov_one_login?
       expect(page).not_to have_link 'Change password'
+      expect(page).to have_content 'This is the name that will appear on your end of module certificate'
       expect(page).to have_content 'Changing your name on this account will not affect your Gov.UK One Login'
```

Comment:
I just changed `Gov` to `GOV` because it started failing. Let me know if that happens on your next run.